dap_aria_mapping/utils/app_data_utils.py

```diff
@@ -31,3 +33,35 @@ def expand_topic_col(topic_df: pl.DataFrame) -> pl.DataFrame:
         (pl.col("topic").str.split("_").arr.get(0)).alias("domain"),
         (pl.col("topic").str.split("_").arr.get(0) + "_" + pl.col("topic").str.split("_").arr.get(1)).alias("area"))
 
+def add_area_domain_chatgpt_names(df_with_topic_ids: pl.DataFrame) -> pl.DataFrame:
+    """adds area, domain, and topic names to a dataframe tagged with topic, area, and domain ids
+
+    Args:
+        df_with_topic_ids (pl.DataFrame): dataframe with id columns named topic, area, domain
+
+    Returns:
+        pl.DataFrame: dataframe tagged with topic names
+    """
+
+    domain_names  = pl.DataFrame(
+        pd.DataFrame.from_dict(
+            get_topic_names("cooccur", "chatgpt", 1, n_top = 35), 
```

Comment:
I apologize for this being a bit messy, but `get_topic_names`should include `postproc` here to fetch the topic names for the postprocessed set of topic names, ie. `get_topic_names("cooccur", "chatgpt", 1, n_top = 35, postproc = True)`. This applies to the other two levels as well.